ste/getProxy_windows.go

```diff
@@ -0,0 +1,35 @@
+package ste
+
+import (
+	"golang.org/x/net/http/httpproxy"
+	"golang.org/x/sys/windows/registry"
+	"net/http"
+	"net/url"
+)
+
+func getProxy() func(*http.Request) (*url.URL, error) {
+	if prox := httpproxy.FromEnvironment(); (prox.HTTPProxy != "" || prox.HTTPSProxy != "") && prox.NoProxy == "" {
+		return http.ProxyFromEnvironment
+		//Option to default to env vars, with a fallback to env vars if grabbing from IE/Edge fails
+	}
+
+	key, err := registry.OpenKey(registry.LOCAL_MACHINE, `Software\Microsoft\Windows\CurrentVersion\Internet Settings`, registry.QUERY_VALUE)
```

Comment:
@adreed-msft It looks like, as is common for many things in the registry, the settings that are actually used are per user. I.e. the ones that get used are in registry.CURRENT_USER, not registry.LOCAL_MACHINE. I verified this just now by changing my own settings, and observing that it was the CURRENT_USER section of the registry that changed, not LOCAL_MACHINE.

Reply:
This would actually make sense-- I was following Microsoft documentation at the time and this was the location they were described to be in; will change this momentarily.